bundlewatch.config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "files": [
```

Comment:
The bundlewatch URL is 404ing because the slug is too long for the Firebase service to handle: [link](https://service.bundlewatch.io/results?d=details%7CrepoOwner%7Ccurology%7CrepoName%7Cradiance-ui%7CrepoCurrentBranch%7Coptimize-rollup%7CrepoBranchBase%7Cmaster%7CcommitSha%7C3d5f0b09d7c175e98ccb23c79004301183d3fd9e%7Cresults%7Cstatus%7Cpass%7CfullResults%7CfilePath%7Cdist%2Fshared-components%2Faccordion%2Findex.cjs%7Csize%7CbaseBranchSize%7CmaxSize%7Cdist%2Fshared-components%2Faccordion%2Fthumbnails%2Findex.cjs%7Cdist%2Fshared-components%2Falert%2Findex.cjs%7Cdist%2Fshared-components%2Favatar%2Findex.cjs%7Cdist%2Fshared-components%2Fbanner%2Findex.cjs%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FanchorLinkButton%2Findex.cjs%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2Fbutton%2Findex.cjs%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FlinkButton%2Findex.cjs%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FroundButton%2Findex.cjs%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FtextButton%2Findex.cjs%7Cdist%2Fshared-components%2Fbutton%2Fshared-components%2Fcontainer%2Findex.cjs%7Cdist%2Fshared-components%2Fbutton%2Fshared-components%2Floader%2Findex.cjs%7Cdist%2Fshared-components%2Fcallout%2Findex.cjs%7Cdist%2Fshared-components%2Fcarousel%2Farrow%2Findex.cjs%7Cdist%2Fshared-components%2Fcarousel%2Findex.cjs%7Cdist%2Fshared-components%2Fcheckbox%2Findex.cjs%7Cdist%2Fshared-components%2Fchip%2Findex.cjs%7Cdist%2Fshared-components%2Fcontainer%2Findex.cjs%7Cdist%2Fshared-components%2FdialogModal%2Findex.cjs%7Cdist%2Fshared-components%2Fdropdown%2Findex.cjs%7Cdist%2Fshared-components%2Ffield%2Findex.cjs%7Cdist%2Fshared-components%2Ficon%2Findex.cjs%7Cdist%2Fshared-components%2FimmersiveModal%2Findex.cjs%7Cdist%2Fshared-components%2Findicator%2Findex.cjs%7Cdist%2Fshared-components%2FloadingSpinner%2Findex.cjs%7Cdist%2Fshared-components%2FoffClickWrapper%2Findex.cjs%7Cdist%2Fshared-components%2FoptionButton%2Findex.cjs%7Cdist%2Fshared-components%2FprogressBar%2Findex.cjs%7Cdist%2Fshared-components%2FradioButton%2Findex.cjs%7Cdist%2Fshared-components%2FselectorButton%2Findex.cjs%7Cdist%2Fshared-components%2Ftabs%2Findex.cjs%7Cdist%2Fshared-components%2Ftoggle%2Findex.cjs%7Cdist%2Fshared-components%2Ftooltip%2Findex.cjs%7Cdist%2Fshared-components%2Ftransitions%2Findex.cjs%7Cdist%2Fshared-components%2FverificationMessages%2FformatMessage%2Findex.cjs%7Cdist%2Fshared-components%2FverificationMessages%2Findex.cjs%7Cdist%2Fshared-components%2Ftypography%2Findex.cjs%7Cdist%2Fshared-components%2Faccordion%2Findex.js%7Cdist%2Fshared-components%2Faccordion%2Fthumbnails%2Findex.js%7Cdist%2Fshared-components%2Falert%2Findex.js%7Cdist%2Fshared-components%2Favatar%2Findex.js%7Cdist%2Fshared-components%2Fbanner%2Findex.js%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FanchorLinkButton%2Findex.js%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2Fbutton%2Findex.js%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FlinkButton%2Findex.js%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FroundButton%2Findex.js%7Cdist%2Fshared-components%2Fbutton%2Fcomponents%2FtextButton%2Findex.js%7Cdist%2Fshared-components%2Fbutton%2Fshared-components%2Fcontainer%2Findex.js%7Cdist%2Fshared-components%2Fbutton%2Fshared-components%2Floader%2Findex.js%7Cdist%2Fshared-components%2Fcallout%2Findex.js%7Cdist%2Fshared-components%2Fcarousel%2Farrow%2Findex.js%7Cdist%2Fshared-components%2Fcarousel%2Findex.js%7Cdist%2Fshared-components%2Fcheckbox%2Findex.js%7Cdist%2Fshared-components%2Fchip%2Findex.js%7Cdist%2Fshared-components%2Fcontainer%2Findex.js%7Cdist%2Fshared-components%2FdialogModal%2Findex.js%7Cdist%2Fshared-components%2Fdropdown%2Findex.js%7Cdist%2Fshared-components%2Ffield%2Findex.js%7Cdist%2Fshared-components%2Ficon%2Findex.js%7Cdist%2Fshared-components%2FimmersiveModal%2Findex.js%7Cdist%2Fshared-components%2Findicator%2Findex.js%7Cdist%2Fshared-components%2FloadingSpinner%2Findex.js%7Cdist%2Fshared-components%2FoffClickWrapper%2Findex.js%7Cdist%2Fshared-components%2FoptionButton%2Findex.js%7Cdist%2Fshared-components%2FprogressBar%2Findex.js%7Cdist%2Fshared-components%2FradioButton%2Findex.js%7Cdist%2Fshared-components%2FselectorButton%2Findex.js%7Cdist%2Fshared-components%2Ftabs%2Findex.js%7Cdist%2Fshared-components%2Ftoggle%2Findex.js%7Cdist%2Fshared-components%2Ftooltip%2Findex.js%7Cdist%2Fshared-components%2Ftransitions%2Findex.js%7Cdist%2Fshared-components%2FverificationMessages%2FformatMessage%2Findex.js%7Cdist%2Fshared-components%2FverificationMessages%2Findex.js%7Cdist%2Fshared-components%2Ftypography%2Findex.js%7Cdist%2Fbundle.js%7Cremoved%7Cdist%2Fbundle-es%2Fshared-components%2Faccordion%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Faccordion%2Fthumbnails%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Falert%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Favatar%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbanner%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbutton%2Fcomponents%2FanchorLinkButton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbutton%2Fcomponents%2Fbutton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbutton%2Fcomponents%2FlinkButton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbutton%2Fcomponents%2FroundButton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbutton%2Fcomponents%2FtextButton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbutton%2Fshared-components%2Fcontainer%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fbutton%2Fshared-components%2Floader%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fcallout%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fcarousel%2Farrow%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fcarousel%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fcheckbox%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fchip%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fcontainer%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FdialogModal%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Fdropdown%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Ffield%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Ficon%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FimmersiveModal%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Findicator%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FloadingSpinner%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FoffClickWrapper%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FoptionButton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FprogressBar%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FradioButton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FselectorButton%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Ftabs%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Ftoggle%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Ftooltip%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Ftransitions%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FverificationMessages%2FformatMessage%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2FverificationMessages%2Findex.js%7Cdist%2Fbundle-es%2Fshared-components%2Ftypography%2Findex.js%7Csummary%7CEverything%2Bis%2Bin%2Bcheck%2B(%252B326.35KB%2C%2B-931.67KB%2C%2B-65.0%2525)%5E377%7C0%7CFSW%7C16Y%7C0%7CFSW%7C4Z7%7C0%7CFSW%7CPD%7C0%7CFSW%7C1HL%7C0%7CFSW%7CRE%7C0%7CFSW%7C3G3%7C0%7CFSW%7C2NW%7C0%7CFSW%7C39W%7C0%7CFSW%7C1FO%7C0%7CFSW%7C385%7C0%7CFSW%7CR2%7C0%7CFSW%7C1S6%7C0%7CFSW%7C1DI%7C0%7CFSW%7C5TY%7C0%7CFSW%7C1Y5%7C0%7CFSW%7C26I%7C0%7CFSW%7C5C%7C0%7CFSW%7C4DL%7C0%7CFSW%7C3II%7C0%7CFSW%7C2P4%7C0%7CFSW%7C1S1%7C0%7CFSW%7C6SZ%7C0%7CFSW%7C1M7%7C0%7CFSW%7C1RN%7C0%7CFSW%7C18M%7C0%7CFSW%7C4AC%7C0%7CFSW%7C25O%7C0%7CFSW%7C1XI%7C0%7CFSW%7C725%7C0%7CFSW%7C1GM%7C0%7CFSW%7C1OM%7C0%7CFSW%7C3TV%7C0%7CFSW%7CVW%7C0%7CFSW%7CQV%7C0%7CFSW%7C1QN%7C0%7CFSW%7C1BOR%7C0%7C270G%7C2O8%7C0%7CFSW%7CV7%7C0%7CFSW%7C46R%7C0%7CFSW%7CFC%7C0%7CFSW%7C14E%7C0%7CFSW%7CFA%7C0%7CFSW%7C2X5%7C0%7CFSW%7C24W%7C0%7CFSW%7C2P0%7C0%7CFSW%7CZA%7C0%7CFSW%7C2V1%7C0%7CFSW%7CNU%7C0%7CFSW%7C1GX%7C0%7CFSW%7C18G%7C0%7CFSW%7C51G%7C0%7CFSW%7C1EJ%7C0%7CFSW%7C1W6%7C0%7CFSW%7C2Z%7C0%7CFSW%7C3N4%7C0%7CFSW%7C2YU%7C0%7CFSW%7C22Y%7C0%7CFSW%7C16G%7C0%7CFSW%7C5OY%7C0%7CFSW%7C1BH%7C0%7CFSW%7C1F1%7C0%7CFSW%7CYU%7C0%7CFSW%7C3KK%7C0%7CFSW%7C1P1%7C0%7CFSW%7C1DT%7C0%7CFSW%7C6E4%7C0%7CFSW%7C11L%7C0%7CFSW%7C1A6%7C0%7CFSW%7C35U%7C0%7CFSW%7CKD%7C0%7CFSW%7CEP%7C0%7CFSW%7C18A%7C0%7CFSW%7C1B4H%7C0%7C270G%7C0%7CH0Q6%7C0%7C0%7C2LC%7C0%7C0%7CV7%7C0%7C0%7C43B%7C0%7C0%7CFC%7C0%7C0%7C11Z%7C0%7C0%7COY%7C0%7C0%7C2Z5%7C0%7C0%7C282%7C0%7C0%7C2TT%7C0%7C0%7C15Q%7C0%7C0%7C2LI%7C0%7C0%7CNU%7C0%7C0%7C1JB%7C0%7C0%7C15V%7C0%7C0%7C4A8%7C0%7C0%7C1PF%7C0%7C0%7C1W6%7C0%7C0%7C2Z%7C0%7C0%7C3ER%7C0%7C0%7C34W%7C0%7C0%7C27O%7C0%7C0%7C1BQ%7C0%7C0%7C5LI%7C0%7C0%7C1BH%7C0%7C0%7C1C6%7C0%7C0%7CYU%7C0%7C0%7C39L%7C0%7C0%7C1NZ%7C0%7C0%7C1K8%7C0%7C0%7C7U5%7C0%7C0%7C11U%7C0%7C0%7C1CA%7C0%7C0%7C39B%7C0%7C0%7CKD%7C0%7C0%7CEP%7C0%7C0%7C1HM%7C0%7C0%7C1CTL%7C0%5E%5E%240%7C%241%7C2%7C3%7C4%7C5%7C6%7C7%7C8%7C9%7CA%5D%7CB%7C%24C%7CD%7CE%7C%40%24F%7CG%7CC%7CD%7CH%7C3Q%7CI%7C3R%7CJ%7C3S%5D%7C%24F%7CK%7CC%7CD%7CH%7C3T%7CI%7C3U%7CJ%7C3V%5D%7C%24F%7CL%7CC%7CD%7CH%7C3W%7CI%7C3X%7CJ%7C3Y%5D%7C%24F%7CM%7CC%7CD%7CH%7C3Z%7CI%7C40%7CJ%7C41%5D%7C%24F%7CN%7CC%7CD%7CH%7C42%7CI%7C43%7CJ%7C44%5D%7C%24F%7CO%7CC%7CD%7CH%7C45%7CI%7C46%7CJ%7C47%5D%7C%24F%7CP%7CC%7CD%7CH%7C48%7CI%7C49%7CJ%7C4A%5D%7C%24F%7CQ%7CC%7CD%7CH%7C4B%7CI%7C4C%7CJ%7C4D%5D%7C%24F%7CR%7CC%7CD%7CH%7C4E%7CI%7C4F%7CJ%7C4G%5D%7C%24F%7CS%7CC%7CD%7CH%7C4H%7CI%7C4I%7CJ%7C4J%5D%7C%24F%7CT%7CC%7CD%7CH%7C4K%7CI%7C4L%7CJ%7C4M%5D%7C%24F%7CU%7CC%7CD%7CH%7C4N%7CI%7C4O%7CJ%7C4P%5D%7C%24F%7CV%7CC%7CD%7CH%7C4Q%7CI%7C4R%7CJ%7C4S%5D%7C%24F%7CW%7CC%7CD%7CH%7C4T%7CI%7C4U%7CJ%7C4V%5D%7C%24F%7CX%7CC%7CD%7CH%7C4W%7CI%7C4X%7CJ%7C4Y%5D%7C%24F%7CY%7CC%7CD%7CH%7C4Z%7CI%7C50%7CJ%7C51%5D%7C%24F%7CZ%7CC%7CD%7CH%7C52%7CI%7C53%7CJ%7C54%5D%7C%24F%7C10%7CC%7CD%7CH%7C55%7CI%7C56%7CJ%7C57%5D%7C%24F%7C11%7CC%7CD%7CH%7C58%7CI%7C59%7CJ%7C5A%5D%7C%24F%7C12%7CC%7CD%7CH%7C5B%7CI%7C5C%7CJ%7C5D%5D%7C%24F%7C13%7CC%7CD%7CH%7C5E%7CI%7C5F%7CJ%7C5G%5D%7C%24F%7C14%7CC%7CD%7CH%7C5H%7CI%7C5I%7CJ%7C5J%5D%7C%24F%7C15%7CC%7CD%7CH%7C5K%7CI%7C5L%7CJ%7C5M%5D%7C%24F%7C16%7CC%7CD%7CH%7C5N%7CI%7C5O%7CJ%7C5P%5D%7C%24F%7C17%7CC%7CD%7CH%7C5Q%7CI%7C5R%7CJ%7C5S%5D%7C%24F%7C18%7CC%7CD%7CH%7C5T%7CI%7C5U%7CJ%7C5V%5D%7C%24F%7C19%7CC%7CD%7CH%7C5W%7CI%7C5X%7CJ%7C5Y%5D%7C%24F%7C1A%7CC%7CD%7CH%7C5Z%7CI%7C60%7CJ%7C61%5D%7C%24F%7C1B%7CC%7CD%7CH%7C62%7CI%7C63%7CJ%7C64%5D%7C%24F%7C1C%7CC%7CD%7CH%7C65%7CI%7C66%7CJ%7C67%5D%7C%24F%7C1D%7CC%7CD%7CH%7C68%7CI%7C69%7CJ%7C6A%5D%7C%24F%7C1E%7CC%7CD%7CH%7C6B%7CI%7C6C%7CJ%7C6D%5D%7C%24F%7C1F%7CC%7CD%7CH%7C6E%7CI%7C6F%7CJ%7C6G%5D%7C%24F%7C1G%7CC%7CD%7CH%7C6H%7CI%7C6I%7CJ%7C6J%5D%7C%24F%7C1H%7CC%7CD%7CH%7C6K%7CI%7C6L%7CJ%7C6M%5D%7C%24F%7C1I%7CC%7CD%7CH%7C6N%7CI%7C6O%7CJ%7C6P%5D%7C%24F%7C1J%7CC%7CD%7CH%7C6Q%7CI%7C6R%7CJ%7C6S%5D%7C%24F%7C1K%7CC%7CD%7CH%7C6T%7CI%7C6U%7CJ%7C6V%5D%7C%24F%7C1L%7CC%7CD%7CH%7C6W%7CI%7C6X%7CJ%7C6Y%5D%7C%24F%7C1M%7CC%7CD%7CH%7C6Z%7CI%7C70%7CJ%7C71%5D%7C%24F%7C1N%7CC%7CD%7CH%7C72%7CI%7C73%7CJ%7C74%5D%7C%24F%7C1O%7CC%7CD%7CH%7C75%7CI%7C76%7CJ%7C77%5D%7C%24F%7C1P%7CC%7CD%7CH%7C78%7CI%7C79%7CJ%7C7A%5D%7C%24F%7C1Q%7CC%7CD%7CH%7C7B%7CI%7C7C%7CJ%7C7D%5D%7C%24F%7C1R%7CC%7CD%7CH%7C7E%7CI%7C7F%7CJ%7C7G%5D%7C%24F%7C1S%7CC%7CD%7CH%7C7H%7CI%7C7I%7CJ%7C7J%5D%7C%24F%7C1T%7CC%7CD%7CH%7C7K%7CI%7C7L%7CJ%7C7M%5D%7C%24F%7C1U%7CC%7CD%7CH%7C7N%7CI%7C7O%7CJ%7C7P%5D%7C%24F%7C1V%7CC%7CD%7CH%7C7Q%7CI%7C7R%7CJ%7C7S%5D%7C%24F%7C1W%7CC%7CD%7CH%7C7T%7CI%7C7U%7CJ%7C7V%5D%7C%24F%7C1X%7CC%7CD%7CH%7C7W%7CI%7C7X%7CJ%7C7Y%5D%7C%24F%7C1Y%7CC%7CD%7CH%7C7Z%7CI%7C80%7CJ%7C81%5D%7C%24F%7C1Z%7CC%7CD%7CH%7C82%7CI%7C83%7CJ%7C84%5D%7C%24F%7C20%7CC%7CD%7CH%7C85%7CI%7C86%7CJ%7C87%5D%7C%24F%7C21%7CC%7CD%7CH%7C88%7CI%7C89%7CJ%7C8A%5D%7C%24F%7C22%7CC%7CD%7CH%7C8B%7CI%7C8C%7CJ%7C8D%5D%7C%24F%7C23%7CC%7CD%7CH%7C8E%7CI%7C8F%7CJ%7C8G%5D%7C%24F%7C24%7CC%7CD%7CH%7C8H%7CI%7C8I%7CJ%7C8J%5D%7C%24F%7C25%7CC%7CD%7CH%7C8K%7CI%7C8L%7CJ%7C8M%5D%7C%24F%7C26%7CC%7CD%7CH%7C8N%7CI%7C8O%7CJ%7C8P%5D%7C%24F%7C27%7CC%7CD%7CH%7C8Q%7CI%7C8R%7CJ%7C8S%5D%7C%24F%7C28%7CC%7CD%7CH%7C8T%7CI%7C8U%7CJ%7C8V%5D%7C%24F%7C29%7CC%7CD%7CH%7C8W%7CI%7C8X%7CJ%7C8Y%5D%7C%24F%7C2A%7CC%7CD%7CH%7C8Z%7CI%7C90%7CJ%7C91%5D%7C%24F%7C2B%7CC%7CD%7CH%7C92%7CI%7C93%7CJ%7C94%5D%7C%24F%7C2C%7CC%7CD%7CH%7C95%7CI%7C96%7CJ%7C97%5D%7C%24F%7C2D%7CC%7CD%7CH%7C98%7CI%7C99%7CJ%7C9A%5D%7C%24F%7C2E%7CC%7CD%7CH%7C9B%7CI%7C9C%7CJ%7C9D%5D%7C%24F%7C2F%7CC%7CD%7CH%7C9E%7CI%7C9F%7CJ%7C9G%5D%7C%24F%7C2G%7CC%7CD%7CH%7C9H%7CI%7C9I%7CJ%7C9J%5D%7C%24F%7C2H%7CC%7CD%7CH%7C9K%7CI%7C9L%7CJ%7C9M%5D%7C%24F%7C2I%7CC%7CD%7CH%7C9N%7CI%7C9O%7CJ%7C9P%5D%7C%24F%7C2J%7CC%7CD%7CH%7C9Q%7CI%7C9R%7CJ%7C9S%5D%7C%24F%7C2K%7CC%7CD%7CH%7C9T%7CI%7C9U%7CJ%7C9V%5D%7C%24F%7C2L%7CC%7C2M%7CH%7C9W%7CI%7C9X%7CJ%7C9Y%5D%7C%24F%7C2N%7CC%7C2M%7CH%7C9Z%7CI%7CA0%7CJ%7CA1%5D%7C%24F%7C2O%7CC%7C2M%7CH%7CA2%7CI%7CA3%7CJ%7CA4%5D%7C%24F%7C2P%7CC%7C2M%7CH%7CA5%7CI%7CA6%7CJ%7CA7%5D%7C%24F%7C2Q%7CC%7C2M%7CH%7CA8%7CI%7CA9%7CJ%7CAA%5D%7C%24F%7C2R%7CC%7C2M%7CH%7CAB%7CI%7CAC%7CJ%7CAD%5D%7C%24F%7C2S%7CC%7C2M%7CH%7CAE%7CI%7CAF%7CJ%7CAG%5D%7C%24F%7C2T%7CC%7C2M%7CH%7CAH%7CI%7CAI%7CJ%7CAJ%5D%7C%24F%7C2U%7CC%7C2M%7CH%7CAK%7CI%7CAL%7CJ%7CAM%5D%7C%24F%7C2V%7CC%7C2M%7CH%7CAN%7CI%7CAO%7CJ%7CAP%5D%7C%24F%7C2W%7CC%7C2M%7CH%7CAQ%7CI%7CAR%7CJ%7CAS%5D%7C%24F%7C2X%7CC%7C2M%7CH%7CAT%7CI%7CAU%7CJ%7CAV%5D%7C%24F%7C2Y%7CC%7C2M%7CH%7CAW%7CI%7CAX%7CJ%7CAY%5D%7C%24F%7C2Z%7CC%7C2M%7CH%7CAZ%7CI%7CB0%7CJ%7CB1%5D%7C%24F%7C30%7CC%7C2M%7CH%7CB2%7CI%7CB3%7CJ%7CB4%5D%7C%24F%7C31%7CC%7C2M%7CH%7CB5%7CI%7CB6%7CJ%7CB7%5D%7C%24F%7C32%7CC%7C2M%7CH%7CB8%7CI%7CB9%7CJ%7CBA%5D%7C%24F%7C33%7CC%7C2M%7CH%7CBB%7CI%7CBC%7CJ%7CBD%5D%7C%24F%7C34%7CC%7C2M%7CH%7CBE%7CI%7CBF%7CJ%7CBG%5D%7C%24F%7C35%7CC%7C2M%7CH%7CBH%7CI%7CBI%7CJ%7CBJ%5D%7C%24F%7C36%7CC%7C2M%7CH%7CBK%7CI%7CBL%7CJ%7CBM%5D%7C%24F%7C37%7CC%7C2M%7CH%7CBN%7CI%7CBO%7CJ%7CBP%5D%7C%24F%7C38%7CC%7C2M%7CH%7CBQ%7CI%7CBR%7CJ%7CBS%5D%7C%24F%7C39%7CC%7C2M%7CH%7CBT%7CI%7CBU%7CJ%7CBV%5D%7C%24F%7C3A%7CC%7C2M%7CH%7CBW%7CI%7CBX%7CJ%7CBY%5D%7C%24F%7C3B%7CC%7C2M%7CH%7CBZ%7CI%7CC0%7CJ%7CC1%5D%7C%24F%7C3C%7CC%7C2M%7CH%7CC2%7CI%7CC3%7CJ%7CC4%5D%7C%24F%7C3D%7CC%7C2M%7CH%7CC5%7CI%7CC6%7CJ%7CC7%5D%7C%24F%7C3E%7CC%7C2M%7CH%7CC8%7CI%7CC9%7CJ%7CCA%5D%7C%24F%7C3F%7CC%7C2M%7CH%7CCB%7CI%7CCC%7CJ%7CCD%5D%7C%24F%7C3G%7CC%7C2M%7CH%7CCE%7CI%7CCF%7CJ%7CCG%5D%7C%24F%7C3H%7CC%7C2M%7CH%7CCH%7CI%7CCI%7CJ%7CCJ%5D%7C%24F%7C3I%7CC%7C2M%7CH%7CCK%7CI%7CCL%7CJ%7CCM%5D%7C%24F%7C3J%7CC%7C2M%7CH%7CCN%7CI%7CCO%7CJ%7CCP%5D%7C%24F%7C3K%7CC%7C2M%7CH%7CCQ%7CI%7CCR%7CJ%7CCS%5D%7C%24F%7C3L%7CC%7C2M%7CH%7CCT%7CI%7CCU%7CJ%7CCV%5D%7C%24F%7C3M%7CC%7C2M%7CH%7CCW%7CI%7CCX%7CJ%7CCY%5D%7C%24F%7C3N%7CC%7C2M%7CH%7CCZ%7CI%7CD0%7CJ%7CD1%5D%5D%7C3O%7C3P%5D%5D).

Is there a reason we're including both `.cjs` and `.js`? I would be comfortable using the larger (`.cjs`) alone, since it should be enough of a smell for `.js` as well.

Reply:
Oh interesting. What if we just do `.js` alone? The `.cjs` bundle is only really for Node where the bundle size isn't as much of an issue. The `.js` bundle is what will be pulled in by Webpack, Rollup, etc.

Reply:
I'm cool with just `.js`.

Reply:
The link is still too long. I think it might be because I moved the folder around so it's trying to send every single file in the URL because they look like they all changed at once. I think it will resolve itself once this is merged and the diff is normal again?

Reply:
Yeah apparently this is a common issue.

https://github.com/bundlewatch/bundlewatch/issues/294
https://github.com/bundlewatch/bundlewatch/issues/34

Reply:
Cool, let’s hope it resolves after merging. Resolving!